test/acl_program_test.cpp

```diff
@@ -1082,11 +1099,9 @@ MT_TEST(from_source, online_mode) {
   ACL_LOCKED(CHECK(acl_realpath_existing(check_str) != ""));
   check_str = m_hashdir + std::string("/build.cmd");
   ACL_LOCKED(CHECK(acl_realpath_existing(check_str) != ""));
-  // Since we invoked compiler with -rtl, we only get an .aoco and .aocr
+  // If we have invoked compiler with -rtl, we should get an .aocr
   check_str = m_hashdir + std::string("/kernels.aocr");
   ACL_LOCKED(CHECK(acl_realpath_existing(check_str) != ""));
-  check_str = m_hashdir + std::string("/kernels/kernels.v");
```

Comment:
Removed this check as we are now just emulating the compile by copying over the precompiled aocr file, and not producing the hardware IP.